### ISSUE TITLE
Zu wenig Fahrzeuge bzw. Zivilisten

### ISSUE THREAD

**Issue by AdlerXiviD** [gruppe-adler/TvT_WITA2016.Altis#31]
Die Anzahl der nutzbaren Fahrzeuge und Zivilisten sind zu gering.

**Comment by TheWillard** [comment]
Die vorhandenen Fahrzeuge waren auch größtenteils abgeschlossen.

**Comment by McDiod** [comment]
Was heißt nutzbar? Die Agents und Bodyguards sollten Lockpicks im Inventar haben.

**Comment by AdlerXiviD** [comment]
Absolut richtig. Dass die Fahrzeuge abgesperrt sind ist kein Problem. Aber die Gesamtanzahl von Fahrzeugen geht auf manchen Karten fast gegen 0.

**Comment by nomisum** [comment]
> 
> 
> Was heißt nutzbar? Die Agents und Bodyguards sollten Lockpicks im Inventar haben.

Stimmt. In einem der letzten Spiele konnte man aus Gründen (Map nicht supported) keine HMMWV kaufen (Alle anderen Fahrzeuge schon, schätze das wird einfach nicht abgefangen wenn er nicht weiß ober woodland nehmen soll oder so), so dass Blufor mit Hellcat only schlechte Karten hatte und auf der Suche nach Fahrzeugen war. Da fiel das insbesondere auf, dass die locked sind und recht wenig in dem Fall.
